io_scene_nif/modules/animation/animation_import.py

```diff
@@ -127,8 +129,9 @@ def import_kf_root(self, kf_root, root):
                 # ignore scale, usually contains invalid data in interpolator
 
             # save priority for future reference
-            # (priorities will be stored into the name of a NULL constraint on
+            # (priorities will be stored into the name of a TRANSFORM constraint on
             # bones, see import_armature function)
+            # This name is a bytestring, not a string
```

Comment:
Should we move it to being a string, to keep things consistent.
Also there is a NULL modifier for the animation, not sure if it would be worth looking into, rather than the proposed direction of using custom properties.